tox.ini

```diff
@@ -3,7 +3,7 @@
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = lint, static-{charm,lib}, unit
+envlist = lint, static-{charm,lib}, unit, integration
```

Comment:
sure that we want to make this default every time one calls tox?

Reply:
What do you mean?

Reply:
just that usually by default the charm tox.ini don't have the integration test running by default, one has to run `tox -e integration`. I guess was made so integration are only run if you really want to run them, and you can run `tox` also outside a juju model deployed.

Reply:
+1, since the integration tests rely on having juju installed and set up, it probably shouldn't be called by default.